### PR TITLE
Create Day3 tasks

### PR DESCRIPTION
**Pull request by KTihirovna**

### CODE REVIEW

File: Day 3 tasks/tasks3.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+ const hoursWorked = prompt("Enter hours:");
+ const ratePerHour = prompt("Enter rate per hour:");
+ const weeklyEarnings = hoursWorked * ratePerHour;
+ alert(`Your weekly earning is ${weeklyEarnings}`);
+ 
+// 9 If the length of your name is greater than 7 say, your name is long else say your name is short.
+// 10. Compare your first name length and your family name length and you should get this output.
+
+let firstName = 'Asabeneh'
+let lastName = 'Yetayeh'
```
Comment (jkalandarov) [inline Day 3 tasks/tasks3.js]: Are you going to do these tasks?

File: Day 3 tasks/tasks3.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+// Your first name, Asabeneh is longer than your family name, Yetayeh
+ 
+
+// 11. Declare two variables myAge and yourAge and assign them initial values and myAge and yourAge.
+  let myAge = 250;
+  let yourAge = 25;
+  alert(`I am ${myAge - yourAge} years older than you.`);
+  
+
+// 12. Using prompt get the year the user was born and if the user is 18 or above allow the user to drive if not tell the user to wait a certain amount of years.
```
Comment (jkalandarov) [inline Day 3 tasks/tasks3.js:160]: Are you going to do this task?

File: Day 3 tasks/tasks3.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+// 14.Create a human readable time format using the Date time object
+
+ /*YYYY-MM-DD HH:mm
+ DD-MM-YYYY HH:mm
+ DD/MM/YYYY HH:mm*/
+
+// Exercises: Level 3
+ //Create a human readable time format using the Date time object. The hour and the minute should be all the time two digits(7 hours should be 07 and 5 minutes should be 05 )
+// YYY-MM-DD HH:mm eg. 20120-01-02 07:05
```
Comment (jkalandarov) [inline Day 3 tasks/tasks3.js]: Are you going to do this task?

File: Day 4 tasks/tasks4.js [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    case 'june':
+    case 'september':
+    case 'november':
+        console.log(`${monthDays} has 30 days.`);
+        break;
+    default:
+        console.log("Invalid month.");
+}
+
+//Write a program which tells the number of days in a month, now consider leap year.
```
Comment (jkalandarov) [inline Day 4 tasks/tasks4.js:198]: Are you going to do this task?

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: Day 3 tasks/tasks3.js (hunk 1 above)
Comment (KTihirovna) [reply Day 3 tasks/tasks3.js]: yes, I am

File: Day 3 tasks/tasks3.js (hunk 2 above)
Comment (KTihirovna) [reply Day 3 tasks/tasks3.js:160]: yes, I' am

File: Day 3 tasks/tasks3.js (hunk 3 above)
Comment (KTihirovna) [reply Day 3 tasks/tasks3.js]: They were some difficult for me. I'm going to do them later

File: Day 3 tasks/tasks3.js (hunk 2 above)
Comment (KTihirovna) [reply Day 3 tasks/tasks3.js:160]: They were some difficult for me. I'm going to do them later

File: Day 4 tasks/tasks4.js (hunk 4 above)
Comment (KTihirovna) [reply Day 4 tasks/tasks4.js:198]: They were some difficult for me. I'm going to do them later

**Review by jkalandarov — APPROVED** [review]

**Pull request merged**